packages/constellation/src/components/Base/Flex/Flex.types.ts

```diff
@@ -0,0 +1,33 @@
+import { CSSProperties, FC } from 'react'
+
+type FlexAlignItems =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline'
+  | 'start'
+  | 'end'
+
+type FlexJustifyContent =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly'
+
+type FlexDirection = 'row' | 'row-reverse' | 'column' | 'column-reverse'
+
+type FlexProps = {
+  alignItems?: FlexAlignItems
+  flexDirection?: FlexDirection
+  gap?: number
+  grow?: boolean
+  justifyContent?: FlexJustifyContent
+  style?: Omit<CSSProperties, 'alignItems' | 'display' | 'flexDirection' | 'gap' | 'justifyContent'>
```

Comment:
Are we allowing arbitrary inline styles in components?

Reply:
For this one the style prop was included in components from the original constellation and I didn't want to change the API